test/expected/triggers-14.out

```diff
@@ -112,7 +112,6 @@ WARNING:  FIRING trigger when: AFTER level: ROW op: INSERT cnt: 3 trigger_name _
 WARNING:  FIRING trigger when: AFTER level: ROW op: INSERT cnt: 3 trigger_name z_test_trigger_all_after
 WARNING:  FIRING trigger when: AFTER level: STATEMENT op: INSERT cnt: 3 trigger_name _0_test_trigger_insert_s_after
 UPDATE hyper SET sensor_1 = 2;
-WARNING:  FIRING trigger when: BEFORE level: STATEMENT op: UPDATE cnt: 3 trigger_name _0_test_trigger_update_s_before
```

Comment:
Isn't it a problem if `_0_test_trigger_update_s_before` and `_0_test_trigger_update_s_after` are never called?

Reply:
Rebase will fix this and not change any trigger behaviour after #4253 is merged.